packages/omuse-framework/pyproject.toml

```diff
@@ -1,2 +1,10 @@
 [build-system]
-requires = [ "setuptools>=42.0.0", "wheel", "amuse-framework" ]
+requires = [ "setuptools>=65.0.0", "wheel", "setuptools_scm", "numpy" ]
```

Comment:
Great that you cleaned this up, but we should probably also stop using setuptools and move to poetry instead.